### PR TITLE
v5.5.0 release prep

### PR DESCRIPTION
**Pull request by shuchitak**

### CODE REVIEW

**Review by danielpieczko — APPROVED** [review]

**Comment by xross** [comment]
Please can you do a few things for me to aid publication?  
[] add archive lib step: https://github.com/xmos/lib_xud/blob/1a25b64107302081e6ff4e8c76a92952ad2c0251/Jenkinsfile#L22
[] make the clones shallow: https://github.com/xmos/lib_xud/blob/1a25b64107302081e6ff4e8c76a92952ad2c0251/Jenkinsfile#L5
[] parameterise INFR_APPS_VERSION: https://github.com/xmos/lib_xud/blob/1a25b64107302081e6ff4e8c76a92952ad2c0251/Jenkinsfile#L49
[] use buildDocs from XJSL

There still seems to be a lot of stuff that needs tidying in this repo. Stuff I noticed: 

[] I think requirements.txt needs moving
[] index.rst in root of repo needs removing 
[] doc structure needs making standard
[] (try and run lib_checks @ develop of infr apps as it will check settings.yml)
[] I think the "script" directory should be "python" - according to other repos
[] examples need renaming to app_
[] is this cmake folder still needed? Its likely confusing for users
[] delete .github/ISSUE_TEMPLATE

I'd like to sort out etc/vanilla and add it to the standard mic_array.h but that might have to wait for now

Sorry quite a bit!

File: doc/exclude_patterns.inc [hunk 1]
```diff
@@ -2,5 +2,4 @@
 /build/*build*/**/*.md
 CHANGELOG.rst
 LICENSE.rst
-**README*
```
Comment (xross) [inline doc/exclude_patterns.inc:5]: good catch

File: settings.yml [hunk 2]
```diff
@@ -1,20 +1,21 @@
+# This file relates to internal XMOS infrastructure and should be ignored by external users
 ---
 lib_name: lib_mic_array
 project: '{{lib_name}}'
-title: '{{lib_name}}: PDM Microphone Arrays'
-version: 5.4.0
+title: '{{lib_name}}: PDM Mic Array library for the XMOS XS3 architecture'
```
Comment (xross) [inline settings.yml:5]: This doesnt match the Readme title

Comment (xross) [reply settings.yml:5]: Should be "PDM microphone array library"

File: doc/rst/lib_mic_array.rst [hunk 3]
```diff
@@ -1,10 +1,11 @@
 .. lib_mic_array documentation master file
 
-Mic Array Library
-=================
+#############
+lib_mic_array
```
Comment (xross) [inline doc/rst/lib_mic_array.rst:4]:
```suggestion
lib_mic_array: PDM microphone array library
```

File: settings.yml (hunk 2 above)
Comment (shuchitak) [reply settings.yml:5]: I've changed the title to PDM microphone array library

File: CHANGELOG.rst [hunk 4]
```diff
@@ -1,6 +1,13 @@
 lib_mic_array change log
 ========================
 
+5.5.0
+-----
+
```
[the remark this answers is not in the document]
Comment (shuchitak) [reply CHANGELOG.rst:6]: done

**Review by xross — APPROVED** [review]
Thanks for your patience!

File: settings.yml (hunk 2 above)
Comment (xross) [reply settings.yml]: Yeah, test needs extending I guess. For new libs id like to make lib_template or similar.

**Pull request merged**